libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/JsCallbackReceiver.java

```diff
@@ -202,7 +202,10 @@ public void executeCallback(String callbackId, String params) {
                 }
                 break;
             case CALLBACK_RESPONSE_STRING:
-                AppLog.d(AppLog.T.EDITOR, callbackId + ": " + params);
+                // log the params unless they contain the post content
+                if (!params.contains("function=getHTMLForCallback~id=zss_field_content~contents=")) {
+                    AppLog.d(AppLog.T.EDITOR, callbackId + ": " + params);
+                }
```

Comment:
does it make sense to add an `else` clause and log something like `params had post content, avoiding logging` just so we can make sure the code actually went through this? As it was acting as a guide when following logs for execution flows

Reply:
Good call, added in 727ce65